auction-app-backend/src/main/java/com/example/auctionapp/controller/BidController.java

```diff
@@ -3,22 +3,43 @@
 import com.example.auctionapp.model.Bid;
 import com.example.auctionapp.request.BidRequest;
 import com.example.auctionapp.service.BidService;
+import com.example.auctionapp.util.SecurityRoles;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import org.springframework.data.domain.Page;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.UUID;
+
 @RestController
 @RequestMapping("/api/v1/bids")
+@SecurityRequirement(name = "JWT Security")
 public class BidController {
     private final BidService bidService;
 
     public BidController(BidService bidService) {
         this.bidService = bidService;
     }
 
-    @PostMapping("/place-bid")
+    @PreAuthorize(SecurityRoles.ALL)
+    @PostMapping(path = "/place-bid")
     public Bid placeBid(@RequestBody final BidRequest bidRequest) {
         return bidService.placeBid(bidRequest);
     }
+
+    @PreAuthorize(SecurityRoles.ALL)
+    @GetMapping(path = "/{userId}")
+    public Page<Bid> getBidsByUserId(
```

Comment:
getBidsForUser or something like that makes more sense than getBidsByUserId, because when we name method, we name them in a way what they do, not how